libpkg/clean_cache.c

```diff
@@ -48,7 +48,7 @@ rm_rf(int basefd, const char *path)
 
 		dirfd = openat(basefd, path, O_DIRECTORY|O_CLOEXEC);
 		if (dirfd == -1) {
-			pkg_emit_errno("openat", path);
+			pkg_emit_errno("openat", __func__);
```

Comment:
Just to pick on the first example I've seen: ```path``` here is the name of the file that ```openat``` is operating on.  How does it make sense to change that to the name of the function where ```pkg_emit_errno()``` is called?